npm/vite-dev-server/src/startServer.ts

```diff
@@ -0,0 +1,52 @@
+import Debug from 'debug'
+import { StartDevServer } from '.'
+import { createServer, ViteDevServer, InlineConfig } from 'vite'
+import { resolve } from 'path'
+import { makeHtmlPlugin } from './makeHtmlPlugin'
+
+const debug = Debug('cypress:vite-dev-server:start')
+
+// TODO: Pull in types for Options so we can infer these
+const serverConfig = (projectRoot: string, supportFilePath: string): InlineConfig => {
+  return {
+    root: resolve(__dirname, projectRoot),
+    base: '/__cypress/src/',
```

Comment:
Need to pull this in from the Cypress options. Currently this is listed as `webpackDevServerPublicPathRoute` 😷

Should rename it and take it in dynamically